src/sentry/api/endpoints/notifications/notification_actions_index.py

```diff
@@ -114,6 +114,8 @@ def get(self, request: Request, organization: Organization) -> Response:
             queryset=queryset,
             on_results=lambda action: serialize(action, request.user),
             paginator_cls=OffsetPaginator,
+            max_per_page=1000,
```

Comment:
Following up on our conversation on Friday, why aren't you doing this in your frontend call? The client can control size of pages using query params.

Reply:
i can't do this in the frontend. if i pass `per_page` it still maxes out at 100 entries

Reply:
i have to specify max_per_page and max_limit on the backend first.
https://github.com/getsentry/sentry/blob/6abb3c6413f6c440f058481072fcb86213cff49d/src/sentry/api/base.py#L513
https://github.com/getsentry/sentry/blob/6abb3c6413f6c440f058481072fcb86213cff49d/src/sentry/api/paginator.py#L260